edb/server/args.py

```diff
@@ -213,6 +214,13 @@ def _writer(status: str) -> None:
              ' Must be an alphanumeric ASCII string, maximum'
              f' {schema_defines.MAX_TENANT_ID_LENGTH} characters long.',
     ),
+    click.option(
+        '--ignore-other-postgres-tenant',
```

Comment:
```suggestion
        '--ignore-other-tenants',
```

Reply:
Should I rename `--postgres-tenant-id` to `--tenant-id` as well?

Reply:
Yes, i see no point in having 'postgres' in those arg names

Reply:
Done.